frontend/lib/main.dart

```diff
@@ -1,9 +1,27 @@
-// start point for dart files
-// I hate workflow setup
-// This is pain
-// Every commit and test prolongs the pain
-// Is it too late to make everything in JS
-// Really hope it works this time
-// MEOW MROW
-// If this works I'm gonna be sad
-// I think I see the light
\ No newline at end of file
+//main.dart
+
+import 'package:flutter/material.dart';
+import 'screens/homescreen.dart';
+import 'screens/secondscreen.dart';
+import 'screens/medicationinfo.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
I like that you've kept the routing on a seperate file, that's generally a good practice